receiver/apachereceiver/testdata/integration/expected.json

```diff
@@ -4,7 +4,7 @@
          "instrumentationLibraryMetrics": [
             {
                "instrumentationLibrary": {
-                  "name": "otel/apache"
+                  "name": "otelcol/apache"
```

Comment:
Would be nice to separate this rename to keep clean git history

Reply:
Sure, I'll remove.